build.gradle.kts

```diff
@@ -0,0 +1,112 @@
+plugins {
+    java
+    `maven-publish`
+}
+
+val javadoc by tasks.existing(Javadoc::class)
+val classes by tasks.existing
+val jar by tasks.existing
+
+group = "io.papermc.lib"
+version = "1.13.1-R0.1-SNAPSHOT"
+
+repositories {
+    mavenCentral()
+    maven("https://papermc.io/repo/repository/maven-public/")
+    maven("https://oss.sonatype.org/content/groups/public/")
+}
+
+dependencies {
+    compileOnly("com.google.code.findbugs:jsr305:1.3.9")
+    compileOnly("com.destroystokyo.paper:paper-api:$version")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+val sourcesJar by tasks.registering(Jar::class) {
+    dependsOn(classes)
+    classifier = "sources"
+    from(sourceSets["main"].allSource)
+}
+
+javadoc {
+    isFailOnError = false
+}
+
+val javadocJar by tasks.registering(Jar::class) {
+    dependsOn(javadoc)
+    classifier = "javadoc"
+    from(javadoc.get().destinationDir)
```

Comment:
`from(javadoc)` should work without the need for `dependsOn(javadoc)`. It definitely works with the old task API, not sure about the lazy one.